### DISCUSSION TITLE
Parse Function To Return CloudwatchLogs event

### DISCUSSION THREAD

**Discussion by spirrello** [awslabs/aws-sdk-rust#669] [Q&A] [answered]
Hello,

Are there plans to write a parse function similar to how the Go SDK has here [here](https://github.com/aws/aws-lambda-go/blob/v1.23.0/events/cloudwatch_logs.go#L22) to return a usable CloudwatchLogs event?

I have looked up and down but can't find something similar for the Rust SDK.

**Comment by Velfi** [comment, marked as answer]
Hey @spirrello, sorry for the lack of response. I think this question is better suited for the folks that maintain [aws-lambda-rust-runtime](https://github.com/awslabs/aws-lambda-rust-runtime). At this time, we have no plans to implement this.
